src/vos/vos_dtx.c

```diff
@@ -657,18 +631,16 @@ dtx_rec_release(struct vos_container *cont, struct vos_dtx_act_ent *dae,
 	if (dae->dae_dbd == NULL)
 		return 0;
 
+	if (dae->dae_oids != NULL && dae->dae_oids != &dae->dae_oid_inline &&
+	    dae->dae_oids != &DAE_OID(dae))
```

Comment:
minor, a helper for the check maybe fine, seems with 3 places in this patch with the same check.

Reply:
One place is for DAE, another two places are for DCE.